mapstruct/pom.xml

```diff
@@ -76,7 +76,7 @@
     </build>
 
     <properties>
-        <org.mapstruct.version>1.4.2.Final</org.mapstruct.version>
+        <org.mapstruct.version>1.5.3.Final</org.mapstruct.version>
```

Comment:
There is any breaking changes?

Reply:
@andrzejsydor No breaking change was encountered. It compiles without any issue and All tests where green.